packages/ui/src/proposals/components/AddProposalButton.tsx

```diff
@@ -1,37 +1,44 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useMemo } from 'react'
 
 import { useApi } from '@/api/hooks/useApi'
 import { TransactionButton } from '@/common/components/buttons/TransactionButton'
 import { PlusIcon } from '@/common/components/icons/PlusIcon'
+import { Tooltip } from '@/common/components/Tooltip'
 import { useFirstObservableValue } from '@/common/hooks/useFirstObservableValue'
 import { useModal } from '@/common/hooks/useModal'
 import { AddNewProposalModalCall } from '@/proposals/modals/AddNewProposal'
 
 export const AddProposalButton = () => {
+  const { api } = useApi()
   const { showModal } = useModal()
-  const addNewProposalModal = useCallback(() => {
-    showModal<AddNewProposalModalCall>({
-      modal: 'AddNewProposalModal',
-    })
-  }, [])
+  const addProposalModal = useCallback(() => showModal<AddNewProposalModalCall>({ modal: 'AddNewProposalModal' }), [])
 
-  const { api } = useApi()
-  const maxProposals = api?.consts.proposalsEngine.maxActiveProposalLimit
-  const currentProposals = useFirstObservableValue(
-    () => api?.query.proposalsEngine.activeProposalCount(),
-    [api?.isConnected]
-  )
-  const areProposalSlotsAvailable = api && maxProposals && currentProposals?.lt(maxProposals)
+  const maxProposals = useMemo(() => api?.consts.proposalsEngine.maxActiveProposalLimit, [api?.isConnected])
+  const proposals = useFirstObservableValue(() => api?.query.proposalsEngine.activeProposalCount(), [api?.isConnected])
+  const availableSlots = useMemo(() => Number(maxProposals) - Number(proposals), [proposals, maxProposals])
+
+  const tooltipProps = useMemo(() => {
+    if (!api?.isConnected) return { tooltipText: 'Connecting to api' }
+    if (availableSlots ?? 0)
+      return {
+        tooltipTitle: 'Max active proposals limit reached',
+        tooltipText: `The creation of proposals is currently disabled because the number of deciding or gracing proposals reached the limit of ${maxProposals}.`,
+        tooltipLinkText: 'Proposal System Constants',
+        tooltipLinkURL: 'https://joystream.gitbook.io/testnet-workspace/system/proposal-system#constants',
+      }
+    return {
+      tooltipText: 'Use the proposal engine to suggest a change to the Council.',
+      tooltipLinkText: 'Learn about the Proposal System',
+      tooltipLinkURL: 'https://joystream.gitbook.io/testnet-workspace/system/proposal-system',
+    }
```

Comment:
Looks like a logic error. This might work:
```suggestion
    if (!api?.isConnected || availableSlots > 0) {
      return {
        tooltipText: 'Use the proposal engine to suggest a change to the Council.',
        tooltipLinkText: 'Learn about the Proposal System',
        tooltipLinkURL: 'https://joystream.gitbook.io/testnet-workspace/system/proposal-system',
      }
    } else {
      return {
        tooltipTitle: 'Max active proposals limit reached',
        tooltipText: `The creation of proposals is currently disabled because the number of deciding or gracing proposals reached the limit of ${maxProposals}.`,
        tooltipLinkText: 'Proposal System Constants',
        tooltipLinkURL: 'https://joystream.gitbook.io/testnet-workspace/system/proposal-system#constants',
      }
    }
```